dts/arm/infineon/psoc6/mpns/CY8C624ABZI_S2D44.dtsi

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2021, Cypress Semiconductor
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <arm/armv7-m.dtsi>
+#include "../psoc6_02/psoc6_02.124-bga.dtsi"
+
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "arm,cortex-m0+";
+			reg = <0>;
+		};
+		cpu@1 {
+			device_type = "cpu";
+			compatible = "arm,cortex-m4f";
+			reg = <1>;
+		};
+	};
+
+	flash-controller@40240000 {
+		compatible = "infineon,mtbhal-flash-controller";
+		reg = < 0x40240000 0x10000 >;
+		#address-cells = <1>;
+		#size-cells = <1>;
+		label = "CYPRESS_FLASH_DRV_NAME";
```

Comment:
why `_DRV_NAME` suffix?

Reply:
good catch. need to be update to something like "PSOC6_FLASH_CONTROLLER"

Reply:
updated